pkg/resource/job.go

```diff
@@ -126,14 +125,24 @@ func (b JobBuilder) MakeContainers() []corev1.Container {
 	// if image == NotSupportedVersion {
 	//		panic("unable to find image")
 	//}
+
 	return []corev1.Container{
 		{
 			Name:            JobContainerName,
 			Image:           b.GetCockroachDBImageName(),
 			ImagePullPolicy: *b.Spec().Image.PullPolicyName,
-			Resources:       b.Spec().Resources,
-			Command:         []string{"/bin/bash"},
-			Args:            []string{"-c", fmt.Sprintf("%s; sleep 150", GetTagVersionCommand)},
+			Resources: corev1.ResourceRequirements{
+				Limits: corev1.ResourceList{
```

Comment:
We can likely make these the same as the requests. This simply runs `cockroach version` and won't require the ability to scale vertically IMO